stormy/models.py

```diff
@@ -9,14 +9,14 @@ class Word(models.Model):
 
 class UserWord(models.Model):
     word = models.ForeignKey(Word, on_delete=models.CASCADE)
-    storm = models.ForeignKey('Storm', on_delete=models.CASCADE)
-    cloud = models.ManyToManyField('Word_Relation')
+    user_storm = models.ForeignKey('Storm', on_delete=models.CASCADE)
+    cloud = models.ManyToManyField('WordRelation')
```

Comment:
this shouldnt be  a many to many with wordrelation, but a many to many with self using a through table
docs: http://docs.djangoproject.com/en/3.1/topics/db/models/#extra-fields-on-many-to-many-relationships
stackoverflow example: https://stackoverflow.com/questions/3880489/how-do-i-write-a-django-model-with-manytomany-relationsship-with-self-through-a